source/includes/container/steps-deploy-minio-single-node-single-drive.rst

```diff
@@ -0,0 +1,159 @@
+     - Directs Podman/Docker to create and start the container as a detached (``-d``) background process with a pseudo-TTY (``-t``).
+       This allows the container to run in the background with an open TTY for bash-like access.
+
+   * - ``-p 9000:9000 -p 9090:9090``
+     - Binds the ports ``9000`` and ``9090`` on the local machine to the same ports on the container.
+       This allows access to the container through the local machine.
+
+   * - ``-v PATH:/data/minio``
+     - Binds the storage volume ``PATH`` on the local machine to the ``/data`` path on the container.
+       Replace this value with the full path to a storage volume or folder on the local machine.
+       For example:
+
+       Linux or MacOS
+         ``~/minio/data/``
+       
+       Windows
+         ``C:\minio\data``
+
+   * - ``-v /etc/default/minio:/etc/config.env``
+     - Mounts the environment file created in the previous step to the ``/etc/config.env`` path on the Container.
+       For Windows hosts, specify the Windows-style path ``-v C:\minio\config:/etc/config.env``.
+         
+       The MinIO Server uses this environment file for configuration.
+         
+   * - ``-e "MINIO_CONFIG_ENV_FILE=/etc/config.env"``
+     - Sets a MinIO environment variable pointing to the container-mounted path of the environment file.
+
+   * - ``--name "minio_local"``
+     - Sets a custom name for the container. 
+       Omit this value to allow Podman/Docker to automatically generate a container name.
+       You can replace this value to best reflect your requirements.
+
+   * - ``minio server --console-address ":9090"``
+     - Starts the MinIO server using the ``minio:minio`` image pulled from an earlier step.
+       The :mc:`minio server --console-address ":9090" <minio server --console-address>` option directs the server to set a static port for the MinIO Console Web Interface.
+       This option is *required* for containerized environments.
+
+       If you modify this value, ensure you set the proper port mapping using the ``-p`` flag to Podman/Docker to ensure traffic forwarding between the local host and the container.
+
+Once you have applied any further customizations to the command, run it in your preferred terminal or shell environment.
+The command should return a unique ID for the created container.
+
+4) Validate the Container Status
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. include:: /includes/container/common-deploy.rst
+   :start-after: start-common-deploy-validate-container-status
+   :end-before: end-common-deploy-validate-container-status
+
+.. code-block:: shell
+
+   Formatting 1st pool, 1 set(s), 1 drives per set.
+   WARNING: Host local has more than 0 drives of set. A host failure will result in data becoming unavailable.
+   MinIO Object Storage Server
+   Copyright: 2015-2022 MinIO, Inc.
+   License: GNU AGPLv3 <https://www.gnu.org/licenses/agpl-3.0.html>
+   Version: RELEASE.2022-07-08T00-05-23Z (go1.18.3 linux/amd64)
```

Comment:
Same question here. Use the global variable from `conf.py`?
Need to consider what happens to line 130 each January.